charts/argo-cd/templates/argocd-server/deployment.yaml

```
@@ -42,10 +42,15 @@ spec:
         imagePullPolicy: {{ default .Values.global.image.imagePullPolicy .Values.server.image.imagePullPolicy }}
         command:
         - argocd-server
-        {{ if .Values.server.staticAssets.enabled }}
+        {{- if .Values.server.staticAssets.enabled }}
         - --staticassets
         - /shared/app
-        {{ end }}
+        {{- else if eq (toString .Values.server.staticAssets.enabled) "<nil>" }}
```

Perhaps something like this could be clearer :thinking: 
```suggestion
        {{- else }}
```

---

if i remove the weird `toString` condition you won't be able to specify `false` without triggering the `semverCompare` clause

maybe the `server.staticAssets.enabled` flag is the bug all along and we can just replace it with just the semver check